docs/dev/dod.md

```diff
@@ -0,0 +1,16 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
+- Travis Build runs without errors (green)
+- Meaningful javadoc comments for:
+  - All classes
+  - All non-private methods and fields
+  - Private methods with non-obvious implementation
```

Comment:
May just request javadoc for all methods, either private or non private, to make things linear